### PR TITLE
Change oneliner to shortened link

### PR DESCRIPTION
**Pull request by chunky-milk**

### CODE REVIEW

**Comment by CleanMachine1** [comment]
Yes and no.

I dislike when links are shortened since you can't see where you are being taken to or what you are downloading
however it does make it look nicer

**Comment by chunky-milk** [comment]
> Yes and no.
> 
> I dislike when links are shortened since you can't see where you are being taken to or what you are downloading
> however it does make it look nicer

True. Also, the user has less characters to type in if they are unable to copy/paste the command.

Also, in case you didn't know, [git.io](https://git.io/) is a link shortening service run by GitHub itself.

**Comment by chunky-milk** [comment]
By the way, this is a great project, keep it up!

**Comment by CleanMachine1** [comment]
with #7 will the link change?

**Comment by CleanMachine1** [comment]
I like it when others commit to my work, makes it all worth it when someone is actually using something I made.

Do you use it?

**Comment by chunky-milk** [comment]
> with #7 will the link change?

No, it won't, it will still be the same location

**Comment by chunky-milk** [comment]
> I like it when others commit to my work, makes it all worth it when someone is actually using something I made.
> 
> Do you use it?

Yes. It's especially helpful when i'm setting up headless servers and copy/pasting installation commands that contain an unnecessary `$` at the beginning.

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ __You need to have **curl** installed to run this command__
 To install, run the following command in a terminal:
 
 ```
-curl -sSl https://raw.githubusercontent.com/CleanMachine1/DollarSkip/master/install.sh | bash
+curl -sSl https://git.io/JsAqw | bash
 ```
```
Comment (CleanMachine1) [inline README.md:22]:
```suggestion
The link is shortened version of the raw file install.sh
```

Can you reword this in anyway?

**Pull request merged**